doc/developer-guides/hld/hostbridge-virt-hld.rst

```diff
@@ -0,0 +1,43 @@
+.. _hostbridge_virt_hld:
+
+Hostbridge emulation
+####################
+
+Overview
+********
+
+Hostbridge emulation is based on PCI emulation. However hostbridge emulation only sets PCI configuration space. Device model set the PCI configuration space for hostbridge in SOS, then expose to UOS to detect the PCI hostbridge.
```

Comment:
@NanlinXie should we start to use the terms `User VM` and `Service VM` now that we're going through these documents?

Reply:
@gvancuts , yes, we need use the terms User VM and Service VM now.

Reply:
Thanks, I have updated it.